keyboards/hyperx/alloy_origins_core/keymaps/default/rules.mk

```diff
@@ -0,0 +1,6 @@
+VIA_ENABLE = no
+
+RGB_MATRIX_ENABLE = yes
+RGB_MATRIX_DRIVER = is31fl3737
+flash_driver = spi
```

Comment:
these can just live inside the `keyboard.json`? They have no place in keymap level